easybuild/easyconfigs/o/OpenMolcas/OpenMolcas-22.06-intel-2022a.eb

```diff
@@ -17,20 +17,21 @@ checksums = [
     '708763e060aa0174bbe9c0cc336b509fbf492066465908dd919452836a7a793a',  # OpenMolcas-21.06_mcpdft_deps.patch
 ]
 
+builddependencies = [('CMake', '3.23.1')]
+
 dependencies = [
     ('GlobalArrays', '5.8.1'),
     ('HDF5', '1.12.2'),
     ('Python', '3.10.4'),
 ]
 
-builddependencies = [('CMake', '3.23.1')]
-
 separate_build_dir = True
```

Comment:
Hmmm forgot this one, this is the default for the cmakemake easyblock, drop it.

Reply:
You mean the `separate_build_dir`, right, not `builddependencies`? https://github.com/easybuilders/easybuild-easyblocks/blob/develop/easybuild/easyblocks/generic/cmakemake.py#L109

Reply:
yes separate_build_dir

Reply:
done